frontend/src/metabase/admin/tasks/containers/TasksApp.tsx

```diff
@@ -0,0 +1,115 @@
+import cx from "classnames";
+import type { ReactNode } from "react";
+import { useState } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+
+import { useListTasksQuery, useListDatabasesQuery } from "metabase/api";
+import AdminHeader from "metabase/components/AdminHeader";
+import PaginationControls from "metabase/components/PaginationControls";
+import Link from "metabase/core/components/Link";
+import AdminS from "metabase/css/admin.module.css";
+import CS from "metabase/css/core/index.css";
+import { Tooltip } from "metabase/ui";
+import type { Database, Task } from "metabase-types/api";
+
+import {
+  InfoIcon,
+  SectionControls,
+  SectionHeader,
+  SectionRoot,
+  SectionTitle,
+} from "./TasksApp.styled";
+
+type TasksAppProps = {
+  children: ReactNode;
+};
+
+export const TasksApp = ({ children }: TasksAppProps) => {
+  const [page, setPage] = useState(0);
+  const { data: tasksData } = useListTasksQuery({
+    limit: 50,
+    offset: page * 50,
+  });
+  const { data: databasesData } = useListDatabasesQuery();
+
+  const tasks = tasksData?.data;
+  const databases = databasesData?.data;
+
+  if (!tasks || !databases) {
+    return null;
```

Comment:
Let's keep error and loading states here too.

Reply:
https://github.com/metabase/metabase/pull/41452/commits/6bc665dd98d5615db51464d788c3cb45436c921c